### PR TITLE
Fixed latitude range clamping on GroundObject. 

### PR DESCRIPTION
**Pull request by dwhipps**
Was incorrectly wrapping to 0..PI and 0..-PI when should have been to PI/2 in both cases.

I added a convenience function:
`CesiumMath called clampToLatitudeRange(angle) `
which clamps the passed value to the correct range, as I feel I've seen this other places and thought others might find it useful.

I added a test to the spec.

### CODE REVIEW

File: Source/Core/Math.js [hunk 1]
```diff
@@ -470,6 +470,27 @@ define([
...
+     * var latitude = Cesium.Math.clampToLatitudeRange(Cesium.Math.toRadians(108.0));
+     */
+    CesiumMath.clampToLatitudeRange = function(angle) {
+        //>>includeStart('debug', pragmas.debug);
+        if (!defined(angle)) {
+            throw new DeveloperError('angle is required.');
+        }
+        //>>includeEnd('debug');
+        
+        return CesiumMath.clamp(angle, -1*CesiumMath.PI_OVER_TWO, CesiumMath.PI_OVER_TWO);
```
Comment (hpinkos) [inline Source/Core/Math.js]: This might be equivalent under the hood anyway, but you can just do `-CesiumMath.PI_OVER_TWO`

**Comment by hpinkos** [comment]
Thanks @dwhipps!  This change looks good to me.  Does anyone else have any input?

File: Source/Core/Math.js (hunk 1 above)
Comment (pjcozzi) [inline Source/Core/Math.js]: +1

File: Source/Core/Math.js [hunk 2]
```diff
@@ -470,6 +470,27 @@ define([
...
+     * Useful for sanitizing data before use in objects requiring correct range.
+     *
+     * @param {Number} angle The latitude value, in radians, to clamp to the range [<code>-Math.PI/2</code>, <code>Math.PI/2</code>).
+     * @returns {Number} The latitude value clamped to the range [<code>-Math.PI/2</code>, <code>Math.PI/2</code>).
+     *
+     * @example
+     * // Clamp 108 degrees latitude to 90 degrees latitude
+     * var latitude = Cesium.Math.clampToLatitudeRange(Cesium.Math.toRadians(108.0));
+     */
+    CesiumMath.clampToLatitudeRange = function(angle) {
```
Comment (pjcozzi) [inline Source/Core/Math.js]: Can you add this to CHANGES.md and add unit tests to [MathSpec.js](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/Specs/Core/MathSpec.js)?

**Comment by pjcozzi** [comment]
@dwhipps one last request, can you please merge in master so we can merge this via GitHub?  There is probably a trivial conflict in CHANGES.md.

**Comment by dwhipps** [comment]
Merged and pushed. Should be good to go. I'd love for this to get into 1.26.

**Comment by pjcozzi** [comment]
@dwhipps are you sure you merged in the lastest `master` branch?  There is still a conflict.

**Comment by dwhipps** [comment]
Take two...

**Comment by pjcozzi** [comment]
Great, thanks again @dwhipps!

**Pull request merged**